src/main/java/com/mineboundteam/minebound/event/GlowingEvent.java

```diff
@@ -0,0 +1,61 @@
+package com.mineboundteam.minebound.event;
+
+import org.lwjgl.glfw.GLFW;
+
+import com.mineboundteam.minebound.MineBound;
+import com.mineboundteam.minebound.registry.KeyRegistry;
+import com.mojang.blaze3d.vertex.PoseStack;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.renderer.OutlineBufferSource;
+import net.minecraft.client.renderer.RenderBuffers;
+import net.minecraft.client.renderer.entity.EntityRenderDispatcher;
+import net.minecraft.util.Mth;
+import net.minecraft.world.entity.Entity;
```

Comment:
Imports on lines 7, 11, and 14 can also be removed as they are not used. 

The imports that were removed did not break anything.

Reply:
EntityRenderDisplatcher and LivingEntity are used in the code, are you sure removing those two won't cause an issue?